code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -412,6 +412,9 @@
 		else
 			CRASH("[src] tried to equip [W] to [slot] in equip_to_slot().")
 
+	var/equipped_to_hands = slot == SLOT_R_HAND || slot == SLOT_L_HAND
+	if(equipped_to_hands)
+		return ITEM_EQUIP_CARRIED
 	if(W.flags_armor_protection)
 		add_limb_armor(W)
```

Comment:
real fix is using /obj/item/equipped() instead of this.

Reply:
Moving these procs to that call instead?

Reply:
yeah along with all the other code that triggers some action on equip/unequip

Reply:
I did try to move this but found it caused more issues, dropped doesn't have the slot so it becomes hard to know whether or not to remove the limb armor.

Since what im adding already existing in the inverse proc (doUnEquip) the equip_to_slot also doing the same feels correct here.

(Unless you have another suggestion)

Reply:
so what you're saying is limb armor is poorly coded